packages/ui/src/lib/Modal.svelte

```diff
@@ -129,16 +135,19 @@
 		}
 	}
 
-	dialog[open]::backdrop {
-		/* NOTE: temporarily hardcoded var(--clr-overlay-bg); */
-		background-color: color(srgb 0 0 0 / 0.34901960784313724);
+	/* backdrop global */
+	/* NOTE: temporarily hardcoded var(--clr-overlay-bg); */
+	:global(dialog[open]::backdrop) {
+		background-color: rgba(214, 214, 214, 0.4);
 		animation: dialog-fade 0.15s ease-in;
 	}
 
-	html.dark dialog[open]::backdrop {
-		/* NOTE: temporarily hardcoded dark var(--clr-overlay-bg); */
-		background-color: color(srgb 0.8392156862745098 0.8392156862745098 0.8392156862745098 / 0.4);
+	/* backdrop dark */
+	/* NOTE: temporarily hardcoded dark var(--clr-overlay-bg); */
```

Comment:
How come these are a) temporary, b) hardcoded?

Reply:
cc @PavelLaptev :)

Reply:
@mtsgrd there was an issue with this `backdrop` thing. it doesn't work with the native `dialog` component. So we're waiting when this bug will be fixed. And this thing doesn't work only on Linux machines by some reason.